journal/202103.md

```diff
@@ -0,0 +1,483 @@
+- The Design [proposal](https://proposals.decred.org/proposals/76eba5a) to cover the remainder of 2021 requests a maximum of $58,850 to cover the remainder of 2021, and reported underspending on the previous proposal such that only 60% of its budget was used - most of the underspend is associated with the identity and visual communication subdomains.
+- The video content [proposal](https://proposals.decred.org/proposals/95a1409) (phase 3) is up for renewal with a maximum budget of $18,800 for another 6 months, and two new contributors (@karamble and @DecredSociety) - for the previous phase only 41% of the budget was used.
+
+high abstain at the start of CV https://matrix.to/#/!uCnoeYfbhUnvwWihzS:decred.org/$iWtdaa0aAZx9_5ntxzcnsiZnUdkTTMPaGW0wuCCa_vo
+
+{ check https://github.com/decredcommunity/proposals/commits/master for updates }
+
+https://decredcommunity.github.io/proposals/approved
+
+design prop report https://github.com/decred/dcrdesign/issues/252
+
+Politeia Digest [issue {}]({}) and [issue {}]({}) have more details on the month's proposals.
+
+## Network
+
+{ section hint: use the same `zoom` parameter for all dcrdata links }
+
+**Hashrate**: March's [hashrate](https://explorer.dcrdata.org/charts?chart=hashrate&zoom={t1}-{t2}&scale=linear&bin=block&axis=time) {set zoom to show the month} opened at ~{n} Ph/s and closed ~{n} Ph/s, bottoming at {n} Ph/s and peaking at {n} Ph/s throughout the month.
+
+Pool hashrate [distribution](https://miningpoolstats.stream/decred) as of {date}: {PoolName n%, ...}.
+
+Distribution of 1,000 blocks [mined](https://miningpoolstats.stream/decred) before {date}: {PoolName n%, ...}.
+
+{ hint: fallback link for pool distribution: https://dcrstats.com/pow }
+
+Hashrate of the previously top UUPool appears to have migrated to Antpool and F2Pool.
+
+**Staking**: [Ticket price](https://explorer.dcrdata.org/charts?chart=ticket-price&zoom={t1}-{t2}&axis=time&visibility=true-true&mode=stepped) varied between {n.n}-{n.n} DCR, with 30-day [average](https://dcrstats.com/) at 177.8 DCR (+{n.n}). { use same t1-t2 for zoom }
+
+The [locked amount](https://explorer.dcrdata.org/charts?chart=ticket-pool-value&zoom={t1}-{t2}&scale=linear&bin=block&axis=time) was {n.nn}-{n.nn} million DCR, meaning that {n.n}-{n.n}% of the circulating supply [participated](https://explorer.dcrdata.org/charts?chart=stake-participation&zoom={t1}-{t2}&scale=linear&bin=block&axis=time) in proof-of-stake.
+
+{ recap of ticket price action if it was interesting }
+
+**Nodes**: Throughout March there were around {n} reachable nodes according to [dcrextdata](https://dcrextdata.planetdecred.org/nodes). Version distribution: {n% app version, ...}.
+
+yes!
+
+- https://nodes.jholdstock.uk/user_agents
+- https://matrix.to/#/!aNnAOHkWUdNcEXRGjJ:decred.org/$eRv4AVR4bKVNyhGpjO__tZ3HYWPJ32f46txaUIB1lpY
+- 386 shot somewhere please
+- is there an API?
+- is code open?
```

Comment:
Code is here https://github.com/jholdstock/dcrmapper

No API at the moment, but trivial to add if somebody wants it.

Reply:
Testnet is available too: https://testnet-nodes.jholdstock.uk

Reply:
> Code is here https://github.com/jholdstock/dcrmapper

I get a 404. Is it public?

Reply:
Oops, try again

Reply:
Got the code, thanks for another cool project! [Added](https://github.com/xaur/decred-news/blob/draft202103/journal/202103.md#dcrmapper) to the draft too.

I'm most interested in tracking user agent counts and how they upgrade. I can get the numbers from the [table](https://nodes.jholdstock.uk/user_agents) quite easily, but wouldn't mind a JSON API with same data.